test-types/describe.test.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @type ./empty.d.ts
+ */
+
+import {
```

Comment:
Lots of automated linter errors in this file.

Reply:
Solved by disabling the rules at `eslintrc.js` as suggested by @SimenB